### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [X] Added required documentation
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: docs/output.md [hunk 1]
```diff
@@ -6,58 +6,221 @@ This document describes the output produced by the pipeline. Most of the plots a
...
 
 </details>
 
-[MultiQC](http://multiqc.info) is a visualization tool that generates a single HTML report summarising all samples in your project. Most of the pipeline QC results are visualised in the report and further statistics are available in the report data directory.
+### Tracks
 
-Results generated by MultiQC collate pipeline QC from supported tools e.g. FastQC. The pipeline has special steps which also allow the software versions to be reported in the MultiQC output for future traceability. For more information about how to use MultiQC reports, see <http://multiqc.info>.
+#### Tracks
+
+Outputs boht junction tracks and bigwig files. For wigToBigWig [`UCSC wigToBigWig`](https://genome.ucsc.edu/goldenPath/help/bigWig.html) is used.
```
Comment (jemten) [inline docs/output.md:95]:
```suggestion
Outputs both junction tracks and bigwig files. For wigToBigWig [`UCSC wigToBigWig`](https://genome.ucsc.edu/goldenPath/help/bigWig.html) is used.
```

File: docs/output.md [hunk 2]
```diff
@@ -6,58 +6,221 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+- `analyse_transcripts`
+  - `*.coverage.gtf`: coverage on the sample.
+  - `*.gene.abundance.txt`: gene abundance on the sample.
+  - `*.transcripts.gtf`: transcripts assembled on the sample
+
+</details>
+
+#### GffCompare
+
+[`GffCompare`](https://github.com/gpertea/gffcompare) annotates stringtie results with the reference marking them each transcript as normal or aberrant.
```
Comment (jemten) [inline docs/output.md:141]:
```suggestion
[`GffCompare`](https://github.com/gpertea/gffcompare) annotates stringtie results with the reference transcripts, marking each assembled transcript as either normal or aberrant.
```

File: docs/output.md [hunk 3]
```diff
@@ -6,58 +6,221 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `analyse_transcripts`
+  - `*.stats`: data summary and accuracy estimation.
+  - `*.annotated.gtf`: annotated gtf file.
+  - `*.tracking`: transcripts assembled on the sample
+  - `*.transcripts.gtf.refmap`: list for each reference transcript what query transcript partially or fully matches it.
+  - `*.transcripts.gtf.tmap`: list the most similar reference trnascript to each query transcript.
```
Comment (jemten) [inline docs/output.md:151]:
```suggestion
  - `*.transcripts.gtf.tmap`: list the most similar reference transcript to each query transcript.
```

File: docs/output.md [hunk 4]
```diff
@@ -6,58 +6,221 @@ This document describes the output produced by the pipeline. Most of the plots a
...
+<summary>Output files</summary>
+
+- `call variants`
+  - `*.vcf.gz`: file in vcf format containing variants found in the patient.
+  - `*.vcf.gz.tbi`: index for .vcf.gz file.
+  - `*.bcftools_stats.txt`: stats on non-reference allele frequency, depth distribution, stats by quality and per-sample counts, singleton stats, etc.
+
+</details>
+
+### Allele specific variant Calling
```
Comment (jemten) [inline docs/output.md:186]:
```suggestion
### Allele specific variant calling
```

**Review by jemten — COMMENTED** [review]
Massive work @Lucpen! 
Some comments and suggestions

File: bin/drop_filter_results.py [hunk 5]
```diff
@@ -76,6 +76,9 @@ def filter_outrider_results(
     df_family_annotated_aberrant_expression_top_hits = annotate_with_hgnc(
         df_family_aberrant_expression_top_hits, out_drop_gene_name
     )
+    df_family_annotated_aberrant_expression_top_hits.to_csv(
+        "OUTRIDER_provided_samples_top_hits.tsv", sep="\t", index=False, header=True
+    )
```
[the remark this answers is not in the document]
Comment (Lucpen) [reply bin/drop_filter_results.py:81]: I will add it to issues :)

**Review by jemten — APPROVED** [review]
🎅 👍

**Pull request merged**